core/src/main/java/com/scalar/db/storage/jdbc/JdbcDatabaseAdmin.java

```diff
@@ -264,7 +264,21 @@ private void createTableInternal(
                 .map(this::enclose)
                 .collect(Collectors.joining(","))
             + "))";
+    if (rdbEngine == RdbEngine.ORACLE) {
+      // For Oracle Database, add ROWDEPENDENCIES to the table to improve the performance
+      createTableStatement += " ROWDEPENDENCIES";
+    }
     execute(connection, createTableStatement);
+
+    if (rdbEngine == RdbEngine.ORACLE) {
+      // For Oracle Database, set INITRANS to 3 and MAXTRANS to 255 for the table to improve the
+      // performance
+      String alterTableStatement =
+          "ALTER TABLE "
+              + enclosedFullTableName(getFullNamespaceName(schemaPrefix, schema), table, rdbEngine)
+              + " INITRANS 3 MAXTRANS 255";
+      execute(connection, alterTableStatement);
+    }
```

Comment:
Should we add the same properties to the metadata table created by the `JdbcTableMetadataManager` ?

Reply:
Thank you for reviewing this!

The properties are for the performance of the parallel transactional execution. So I don't think we need the properties to the metadata.

Reply:
All right!